CI fails with `nuget restore`

For example: CI #<span/>1196 Attempt #<span/>2
https://github.com/y-iihoshi/ThScoreFileConverter/runs/6432659306?check_suite_focus=true#step:6:8
```
Run nuget restore ThScoreFileConverter.sln
  nuget restore ThScoreFileConverter.sln
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\3.10.4\x64
MSBuild auto-detection: using msbuild version '17.2.1.25201' from 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\bin'.

MsBuild timeout out while trying to get project to project references.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -40,12 +40,9 @@ jobs:
     - name: Add msbuild to PATH
       uses: microsoft/setup-msbuild@v1.1
 
-    - name: Restore NuGet packages
-      run: msbuild -t:restore
-
     - name: Build
       run: >
-        msbuild ThScoreFileConverter.sln /m /verbosity:minimal
+        msbuild ThScoreFileConverter.sln /m /restore /verbosity:minimal
         /p:Configuration=${{ matrix.configuration }}
         /p:TargetFrameworks=${{ matrix.target-framework }}
 

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -37,11 +37,11 @@ jobs:
           ${{ runner.os }}-nuget-${{ matrix.target-framework }}-
           ${{ runner.os }}-nuget-
 
-    - name: Restore NuGet packages
-      run: nuget restore ThScoreFileConverter.sln
-
     - name: Add msbuild to PATH
       uses: microsoft/setup-msbuild@v1.1
+
+    - name: Restore NuGet packages
+      run: msbuild -t:restore
 
     - name: Build
       run: >

